AlphaWallet/AppCoordinator.swift

```diff
@@ -357,6 +357,11 @@ extension AppCoordinator: CustomUrlSchemeCoordinatorDelegate {
 }
 
 extension AppCoordinator: AssetDefinitionStoreCoordinatorDelegate {
+
+    func didHandleOpenURL(in coordinator: AssetDefinitionStoreCoordinator, with result: OpenURLResult) {
```

Comment:
It seems like there is already a `show(error:for:)` function in `AssetDefinitionStoreCoordinatorDelegate` (currently not called by the delegate) which we can call when there's an error.

There's also `addedTokenScript(forContract:, forServer:)` for when the override is successful.

So you can probably remove `didHandleOpenURL(in:with:)`, use those 2 instead?